supabase/client.py

```diff
@@ -152,10 +153,14 @@ def _init_supabase_auth_client(
 
     @staticmethod
     def _init_postgrest_client(
-        rest_url: str, supabase_key: str, headers: Dict[str, str], schema: str
+        rest_url: str, 
+        supabase_key: str, 
+        headers: Dict[str, str], 
+        schema: str,
+        timeout: Union[int, float, Timeout] = DEFAULT_POSTGREST_CLIENT_TIMEOUT
```

Comment:
[httpx source doesn't seem to include int as an expected timeout type](https://github.com/encode/httpx/blob/67e9c14952340bfd799a0de41dfaf269f376653d/httpx/_types.py#L63) -- are you sure ints work fine here?

Reply:
Yeah it does, and besides it sort of mirrors the `postgrest`'s code [here ](https://github.com/supabase-community/postgrest-py/blob/09142ab374f51a5ff7b339f071fc8d369c14f144/postgrest/_sync/client.py#L27)